eox_tenant/receivers_helpers.py

```diff
@@ -0,0 +1,49 @@
+"""
+Common utilities for use along signals file.
+"""
+
+from eox_tenant.models import TenantConfig, Microsite
+
+
+def get_tenant_config_by_domain(domain):
+    """
+    Reach for the configuration for a given domain.
+
+    **Arguments**
+        domain: String parameter.
+
+    **Returns**
+        configurations: dict
+        external_key: String
+
+        **Example**
```

Comment:
Is this config example really necessary, I think by now somebody reading the code know what to expect in the configuration dict? maybe I'm too used to this, but still